WPB-11617: Doument the Setup a scoped user for cargohold (#77)
wireapp/wire-docs: src/how-to/install/ansible-VMs.md

- In your ‘hosts.ini’ file, in the `[all:vars]` section, make sure
  you set the ‘minio_network_interface’ to the name of the interface
  you want minio nodes to talk to each other on. The default from the
  playbook is not going to be correct for your machine.

```ini
[all:vars]
# Default first interface on ubuntu on kvm:
minio_network_interface=ens3
```

#### Configure Access Key and Secret Key for MinIO and Cargohold Service

> **Purpose**: Configure secure, least-privilege access for the Cargohold service to use MinIO object storage.

Reviewer: ```suggestion
**Purpose**: Configure a secure, least-privilege access method for the Cargohold service to utilize the MinIO object storage.
```

**Security Model**:
- **MinIO root credentials**: Used only for administrative purposes

> - **Cargohold IAM user**: Least privileged user with policy access only to the `assets` bucket

Reviewer: ```suggestion
- **Cargohold IAM user**: A least privileged user with a policy that only gives access to the `assets` bucket
```

- **Service account**: Separate access/secret key pair for Cargohold service operations

## Setup Process

1. **Generate credentials**: Run `./bin/offline-secrets.sh` from the wire-server-deploy directory.

   This generates a `secrets.yaml` file in `ansible/inventory/offline/group_vars/all/` with:
   ```yaml
   minio_access_key: "<MINIO ROOT ACCESS KEY>"
   minio_secret_key: "<MINIO ROOT SECRET KEY>"
   minio_cargohold_access_key: "<MINIO CARGOHOLD ACCESS KEY>"
   minio_cargohold_secret_key: "<MINIO CARGOHOLD SECRET KEY>"
   ```

> 2. **For existing Wire systems** - backup and regenerate secrets:

Reviewer: ```suggestion
2. **For existing Wire systems** - Backup and regenerate secrets:
```

   ```bash
   # Backup current secrets file
   cp ansible/inventory/offline/group_vars/all/secrets.yaml \
      ansible/inventory/offline/group_vars/all/secrets.yaml.backup

   # Remove current secrets and generate new ones
   rm ansible/inventory/offline/group_vars/all/secrets.yaml
   ./bin/offline-secrets.sh
   ```

> 3. **Migration step**: Replace the newly generated `minio_access_key` and `minio_secret_key` with the values from `secrets.yaml.backup` to maintain compatibility.

Reviewer: this is unclear, and could mean merge the stuff from the backup into the new secret, or vica-versa.
